calculator/src/components/calculator.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import {evaluate} from "mathjs";
+import NumberFormat from 'react-number-format'
+import '../css/calculator.css'
+
+export default class Calculator extends Component {
+
+    constructor(props) {
+        super(props);
+    
+        this.state = {
+          input: ""
```

Comment:
It could be helpful to also keep track of the previous value. That would solve the issue of the two value on display you have right now.